LucasTerra.lua

```diff
@@ -2559,27 +2559,37 @@ end
 -- @param	name¹, name², name*, ...	The names to add on safe list.
 -- @returns void
 
-function addtosafelist(safetype, ...)
-	local alarmtypes = {"PlayerOnScreen", "PlayerAttacking", "MonsterAttacking", "PrivateMessage", "DefaultMessage", "GMDetected", "Disconnected", "CharacterStuck", "HealthBelow", "ManaBelow", "UnjustKill", "EnemiesOnline"}
-	local alarmtypesLower = alarmtypes
-	table.tolower(alarmtypesLower)
-	local pos = table.find(alarmtypesLower,alarmtype:lower())
-	if not pos then
+function addtosafelist(safeType, ...)
+	local alarmTypes = {"PlayerOnScreen", "PlayerAttacking", "MonsterAttacking", "PrivateMessage", "DefaultMessage"}
+	safeType = safeType:lower()
+
+	local foundAlarmType = nil
+	for _, v in ipairs(alarmTypes) do
+		if safeType == v:lower() then
+			foundAlarmType = v
+			break
+		end
+	end

```

Comment:
Because I'd have to use table.lower on alarmTypes and if i do, the path will be broken.
What i mean is you cannot have paths like this: /playeronscreen/PlaySound, the letter case is important here.